file-formats/tabl/type/zif_aff_indx_v1.intf.abap

```diff
@@ -0,0 +1,71 @@
+INTERFACE zif_aff_indx_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Index Fields</p>
+    "! Index fields
+    BEGIN OF ty_indexfield,
+      "! <p class="shorttext">Field Name</p>
+      "! Field name
+      "! $required
+      field_name TYPE c LENGTH 30,
+    END OF ty_indexfield,
+    tt_index_fields TYPE TABLE OF ty_indexfield WITH DEFAULT KEY
+      WITH UNIQUE SORTED KEY name_unique COMPONENTS field_name.
+
+  TYPES:
+    "! <p class="shorttext">ABAP DDIC Secondary Index Properties</p>
+    "! Object type INDX
+    BEGIN OF ty_main,
+      "! $required
+      format_version        TYPE if_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                TYPE if_aff_types_v1=>ty_header_60,
```

Comment:
Do we need a full-blown `header`? I guess the object needs only a description. Therefore, you might want to change the type to `zif_aff_types_v1=>ty_header_only_description`

```suggestion
      header                TYPE zif_aff_types_v1=>ty_header_only_description,
```

Reply:
As discussed, we need a full-blown header with abap_language_version. No change here.

Reply:
I agree, for R3TR XINX, you will need it. 
But for LIMU objects, I expect the ABAP language version and the original language are specified for the table.

I would create different headers for R3TR XINX and LIMU XINX